core/src/main/java/org/apache/brooklyn/entity/group/SshCommandMembershipTrackingPolicy.java

```diff
@@ -61,11 +64,16 @@
  * executed. This can be one of: the {@link ExecutionTarget#ENTITY owing entity};
  * the {@link ExecutionTarget#MEMBER member} that was updated; or
  * {@link ExecutionTarget#ALL_MEMBERS all members} of the group.
+ * 
+ * @deprecated introduced and removed in 0.10.0-snapshot; use a combination of 
+ * InvokeEffectorOnSensorChange and SshCommandEffector instead
+ * (much simpler semantics as you normally will want to listen to a local sensor
+ * (which in turn is aggregated over members) instead of children sensors directly).
+ * If there is a need for {@link #EVENT_TYPE} or {@link #MEMBER_ID} then we'll want
+ * InvokeEffectorOnSensorChange to be able to subscribe to members/children etc as well,
+ * and pass parameters based on source.entityId and source.sensor.
  */
-// TODO might make sense to split up behaviour into two classes,
-// an InvokeEffectorMembershipTrackingPolicy and an SshMultiEntityCommandEffector -- 
-// where the latter has the configurable target introduced here
-@Beta
+@Deprecated
```

Comment:
Actually, this could be safely removed, since the only usage of it has been updated to use the new policy defined here.